.circleci/config.yml

```diff
@@ -172,6 +172,17 @@ jobs:
       - image: mcr.microsoft.com/dotnet/core/sdk:2.2
     steps:
       - checkout
+      - run:
+          name: Install tools
+          command: |
+            apt-get update
+            apt-get install -y libxml2-utils
+            dotnet tool install -g trx2junit
+      - run:
+          name: Add local nuget configuration
```

Comment:
concerned about the fragility here, as this is the only job that performs this step. whenever we get rid of the Logging project, someone will have to remember/know that this is the only job that performs this step and it can get lost easily. may be worth putting in every C# job to be more resilient